tests/fuzz/runFuzzTest.sh

```diff
@@ -262,5 +244,23 @@ then
 else
     cat ${TEST_NAME}_output.txt
     printf "\033[31;1mFAILED\033[0m %s, %6d features covered\n" "$TEST_INFO" $FEATURE_COVERAGE
+    
+    # Store corpus to S3 to be used for debugging if the test fails
+    unset LD_PRELOAD
+    unset LD_LIBRARY_PATH
+    if [ "$CORPUS_UPLOAD_LOC" != "none" ]; then
+        printf "Zipping corpus files...\n"
+        zip -r ./corpus/${TEST_NAME}.zip ./corpus/${TEST_NAME}/
+        
+        printf "Uploading zipped corpus file to S3 bucket...\n"
+        aws s3 cp ./corpus/${TEST_NAME}.zip $CORPUS_UPLOAD_LOC/${TEST_NAME}/corpus_$(date +%Y-%m-%d-%T).zip
+    fi
+
+    # Store generated output files in the S3 bucket.
+    if [ "$ARTIFACT_UPLOAD_LOC" != "none" ]; then
```

Comment:
Curious where `$ARTIFACT_UPLOAD_LOC` is getting set? In the job definition?

Reply:
Yes, it's defined by environment variable in the job.